metadata-ingestion/docs/sources/dremio/dremio_recipe.yml

```diff
@@ -0,0 +1,29 @@
+source:
+  type: dremio
+  config:
+    # Coordinates
+    hostname: localhost
+    port: 9047
+    tls: true
+
+    # Credentials with basic auth
+    authentication_method: password
+    username: user
+    password: pass
+
+    # Credentials with personal access token
+    authentication_method: PAT
+    password: pass
+
+    include_query_lineage: True
+
+    source_mappings:
+      - platform: s3
+        platform_name: samples
```

Comment:
```suggestion
        source_name: samples
```

Reply:
Make source_name and platform mandetory if source_mapping is provider